snuba/migrations/snuba_migrations/events/0014_backfill_errors.py

```diff
@@ -206,3 +200,24 @@ def backfill_errors() -> None:
                 PARTITION {partition} FINAL DEDUPLICATE
                 """
             )
+
+
+class Migration(migration.CodeMigration):
+    """
+    Backfills the errors table from events.
+    Errors replacements should be turned off while this script is running.
+    Note this migration is not reversible.
+    """
+
+    blocking = True
+
+    def forwards_global(self) -> Sequence[operations.RunPython]:
+
+        return [
+            operations.RunPython(
```

Comment:
IS there a way we can provide some progress indicators to the client around where we are in the migration? For a big table this will be there waiting for a long time without giving any indication on what is happening.

Reply:
It can be estimated by getting a count of events to migrate before starting.

Reply:
Since we print out logging messages in this migration already as we do not have a logger, what about periodically printing out the progress?  It should be easy as the script is anyway breaking down the migration time windows of time instead of running one massive statement.

I doubt a lot of people would have the instinct of checking the size of the table before running the migration.

Reply:
Will add